locale/fi/combobox.json

```diff
@@ -0,0 +1,58 @@
+{
+  "title": "Monivalinta (Combobox)",
+  "intro": "Monivalinta-komponentin avulla käyttäjä voi valita yhden tai useamman vaihtoehdon listalta.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Listalla olevien vaihtoehtojen tekstien olisi hyvä olla uniikkeja.",
```

Comment:
This I would change so that all items *have* to be unique rather than just *should* be.

Reply:
It was wanted that texts could be same and therefore the `uniqueItemId` prop was introduced.
But maybe it is still wise to "force" it more for them to be unique texts.

Reply:
This is now changed to be understood more like "have to be", to guide users to use it correctly UX-way.